pyspecProcScripts/correlation_alignment.py

```diff
@@ -36,9 +36,10 @@ def correl_align(
 
     Parameters
     ==========
-    s_orig:  psd.nddata
+    s_jk:  psd.nddata
```

Comment:
this should still be called s_orig, I think.  I think the strategy here was to make a copy (once!) so the original data is unmodified.